scipy/stats/stats.py

```diff
@@ -3712,7 +3750,14 @@ def __init__(self, msg=None):
         self.args = (msg,)
 
 
-def pearsonr(x, y):
+ConfidenceInterval = namedtuple('ConfidenceInterval', ['low', 'high'])
+
+PearsonRResult = _make_tuple_bunch('PearsonRResult',
+                                   ['r', 'pvalue'],
+                                   ['fishers_ci'])
+
+
+def pearsonr(x, y, *, alternative='two-sided', confidence_level=0.95):
```

Comment:
Why do we require that additional arguments be keyword only? Something I should start doing?

Reply:
For API development, this ensures that we can later add a new keyword parameter without requiring it to be the last parameter in the definition.  On an esthetic level, it forces users to write more readable code.

We haven't formally incorporated this into a recommend coding style, but for functions with more than two or three parameters, I think it would be a good idea.